### PR TITLE
#5289 Reuse name of dissolved account.

### PR DESCRIPTION
**Pull request by mengdong19**
*Issue #:5289
https://github.com/bcgov/entity/issues/5289

*Description of changes:*

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [X ] No lint errors
- [X ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/src/auth_api/resources/org.py [hunk 1]
```diff
@@ -85,12 +85,17 @@ def get():
...
         limit = request.args.get('limit', 10)
+        validate_name = request.args.get('validateName', 'False')
 
         try:
             token = g.jwt_oidc_token_info
-            response, status = OrgService.search_orgs(business_identifier=business_identifier, access_type=access_type,
-                                                      name=name, status=status, bcol_account_id=bcol_account_id,
-                                                      page=page, limit=limit, token=token), http_status.HTTP_200_OK
+            if validate_name.upper() == 'TRUE':
+                response, status = OrgService.find_by_org_name(name), http_status.HTTP_200_OK
```
Comment (saravanpa-aot) [inline auth-api/src/auth_api/resources/org.py:93]: If we return the full org , that could cause a problem.. anyone can access the orgs details using this API if they know the name...
Can we not return the full org details ? may be just return the status Or just the name

File: auth-api/requirements.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ Mako==1.1.3
 MarkupSafe==1.1.1
 SQLAlchemy-Continuum==1.3.11
 SQLAlchemy-Utils==0.36.8
-SQLAlchemy==1.3.19
+SQLAlchemy==1.3.20
```
Comment (nitheesh-aot) [inline auth-api/requirements.txt:12]: are these upgrades in requirements.txt are ok?

@sumesh-aot @saravanpa-aot

Comment (saravanpa-aot) [reply auth-api/requirements.txt:12]: yup

File: auth-api/src/auth_api/resources/org.py (hunk 1 above)
Comment (mengdong19) [reply auth-api/src/auth_api/resources/org.py:93]: It doesn’t rerun any information. Just blank one or none. The same as before.

Comment (saravanpa-aot) [reply auth-api/src/auth_api/resources/org.py:93]: but its returning the OrgService.find_by_org_name(name) ? rite?

Comment (saravanpa-aot) [reply auth-api/src/auth_api/resources/org.py:93]: I will just check the response of this call; if it doesnt have any org information ; I am good

Comment (mengdong19) [reply auth-api/src/auth_api/resources/org.py:93]: Response from service is processed at resource, based on role. Public user only see 200 and blank content {}, or 204 no content.

**Review by saravanpa-aot — APPROVED** [review]

File: auth-api/src/auth_api/resources/org.py (hunk 1 above)
Comment (saravanpa-aot) [reply auth-api/src/auth_api/resources/org.py:93]: thats good then

**Review by saravanpa-aot — COMMENTED** [review]
guess its getting more changes or just merge problems?

**Review by saravanpa-aot — APPROVED** [review]

**Pull request merged**